<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatisticAnalysis</class>
 <widget class="QWidget" name="StatisticAnalysis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="1,20" columnstretch="0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1,0">
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>日报</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>周报</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>月报</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="dateTimeEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnquery">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>1</width>
         <height>1</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
